enhancements/operator-hub-check.md

```diff
@@ -0,0 +1,217 @@
+---
+title: operatorhub-bundle-validation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-19
+last-updated: 2021-03-07
+status: implementable
+---
+
+# Add checks new Validator
+
+This proposal describes new checks to be implemented into the [OperatorHubValidator][operatorhubvalidator]. This validator is defined in [operator-framework/api][oper-api].
+
+## Open Questions
+
+**1. Should we create a new Validator for this new checks ?** 
+
+The new checks are valid in the context of what would be required and/or recommended to publish an operator bundle on OperatorHub.io. Also, we do not have many of them to starts to have big concerns which shows fail in the [PO](https://wiki.c2.com/?PrematureOptimization) in this stage.
+
+However, would be expected following good practices where the types of checks are properly encapsulate and can easily keep maintained and extracted if required.
+
+**2. Are ONlY these checks enough to ensure that the operator bundles work well as expected?** 
+
+Definitely not. The operator bundle manifests has a lot of information which could be used by *"linters"* checks as this one with the purpose of avoid issues scenarios and give a fast response for the operator authors and tools.
+
+This proposal only address some priority needs which are; 
+- check and notify the main recommendations for us at the moment; [channel naming convention][channel-names-doc] and operator name versioning.
+- avoid the issues with the most critical scenarios related to K8S API deprecations which will cause high impact after 1.22 release out.
+
+**3. What are the changes proposed here which affects SDK?** 
+
+This proposal is to define new checks for a Validator which is used and provided by SDK already. Then, for users are able to use its new checks SDK requires only bump the version of [operator-framework/api][oper-api] and add a new option flag `--optional-values` to get and pass options values to the Validator. 
```

Comment:
@jmrodri in order to make clear it,  I added the info via the open question ^.

Reply:
the idea here is only to provide a flag which is a string map "key=value,key=value" for the CLI pass options values for the API. The SDK does not require to validate that. If the value informed be invalid then the API will return the error and/or ignore it.  

WHY does it need to be a map? 

Because if we want to add other values then we can just use it. Also, this option flag can be used with any other validator. For example, if the OCP one gets done as we have the OperatorHubValidator today than, we will pass ocp=4.8 for example.

Reply:
Added on the proposal. So, I am closing this one as sorted. However, please feel free to ping me if think that requires a better description.